integrations/totango/lib/index.js

```diff
@@ -44,10 +44,14 @@ Totango.prototype.initialize = function(page) {
     track:function(t,o,n,a){window.totango_tmp_stack.push({activity:t,module:o,org:n,user:a}); return -1;}};
   /* eslint-enable */
 
+  /* Int Totango JS */
+  var defaultRegion = 'us'
+  var selected_region = this.options.region ?  this.options.region : defaultRegion; 
```

Comment:
@shmulikgolan please define the `region` option on the code block of line 17-25. You can also default the option to `us` on the same code block rather than doing it on line 48.